src/core/timer.c

```diff
@@ -391,12 +393,18 @@ static void timer_enter_waiting(Timer *t, bool time_change) {
                 if (v->base == TIMER_CALENDAR) {
                         usec_t b, rebased;
 
-                        /* If we know the last time this was
-                         * triggered, schedule the job based relative
-                         * to that. If we don't, just start from
-                         * the activation time. */
-
-                        if (dual_timestamp_is_set(&t->last_trigger))
+                        /* If DeferReactivation is enabled, schedule
+                         * the job based on the last time the trigger unit
+                         * entered inactivity. Otherwise, if we know the
+                         * last time this was triggered, schedule the job
+                         * based relative to that. If we don't, just start
+                         * from the activation time or realtime. */
```

Comment:
btw, we line break to 109ch these days by default. see .editorconfig and similar files. some old code retains other line breaks, but please adjust them as you touch code.

Reply:
Unaddressed.

Reply:
Still not?? Please do not break lines so frequently.

Reply:
Sorry, I thought that you were referring to the line below, which was *over* 109 ch